identity-apps-core/apps/recovery-portal/src/main/resources/org/wso2/carbon/identity/mgt/recovery/endpoint/i18n/Resources.properties

```diff
@@ -335,3 +335,7 @@ QmFja3VwIENvZGUgRW5hYmxlZA__=Backup Code Enabled
 RW5hYmxlZCBBdXRoZW50aWNhdG9ycw__=Enabled Authenticators
 QmFja3VwIENvZGVz=Backup Codes
 RmFpbGVkIEJhY2t1cCBDb2RlIEF0dGVtcHRz=Failed Backup Code Attempts
+invitation.accepted=Invitation Accepted
```

Comment:
Shall we add the i18n strings under the correct category in the file?

Reply:
These are new i18n which are not clearly seperated to the existing categories. Hense we will keep these as it is.